.github/workflows/test.yml

```diff
@@ -0,0 +1,357 @@
+name: Test all packages
+
+on:
+  pull_request:
+    types:
+      - synchronize
+      - opened
+      - reopened
```

Comment:
There is a question. Do we need trigger tests on PR open, reopen? I mean these tests are strictly related to the source code change 🤔

Reply:
You're probably right 👍